src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -239,9 +239,9 @@ class ChannelData {
   void DestroyResolvingLoadBalancingPolicyLocked();
 
   static bool ProcessResolverResultLocked(
-      void* arg, const Resolver::Result& result, const char** lb_policy_name,
+      void* arg, const Resolver::Result& result,
```

Comment:
It looks like this is also including the changes from #22100.  That's probably fine, though; let's just note it in the PR description.